delta_shooter/game_mechanics.py

```diff
@@ -0,0 +1,310 @@
+import random
```

Comment:
Comment shouldn't be on this line, but it doesn't let me comment on `check_submission.py` - it's still a placeholder